boltons/funcutils.py

```diff
@@ -495,22 +496,26 @@ def __init__(self, name, **kw):
     # def get_argspec(self):  # TODO
 
     if _IS_PY2:
-        def get_sig_str(self):
+        def get_sig_str(self, with_annotations=True):
```

Comment:
not sure if this flag is all that helpful here. not used in the body, and there's no argument for it in inspect.formatargspec on py2

Reply:
if the goal is to make it forward compatible for py2/3 users, that's fine, maybe just a docstring letting them know it's ignored on py2 :)

Reply:
Went with removing it, since the flag is really only useful for internal implementation reasons, no reason for external caller to ever set it.